pages/store.php

```diff
@@ -62,10 +62,24 @@ function store() {
 				throw new Exception("You already own this item!");
 			}
 			
+			$max_missing = -1;
+			if (isset($_GET['max'])) {
```

Comment:
Could we move all the new logic into this if block? I think it would be easier to maintain if we have all the logic for purchasing max items and all the logic for purchasing one item isolated in their own logic blocks. 

As is I'm concerned that it's not clear what values of `max_missing` can cause a user to buy multiple consumables, with the multiple logic blocks below interacting with it